### PR TITLE
Add basic Theme to Home and Products

### PR DESCRIPTION
**Pull request by Raghu-Ch**

### CODE REVIEW

File: public/index.css [hunk 1]
```diff
@@ -49,28 +64,16 @@ footer {
 
 #contact>a {
   text-decoration: none;
-  color: #9d9d9d;
+  color: white;
```
Comment (smandada) [inline public/index.css]: use hex code

File: public/index.css [hunk 2]
```diff
@@ -1,29 +1,44 @@
 
 /*
+   @ Index Styles
+*/
 
-  @ Styles
+body {
+  background: linear-gradient(to right, #f7ce40, white);
```
Comment (smandada) [inline public/index.css]: use hex-code

File: public/beardinfo/beardinfo.html [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+            </h2>
             <h4>
                 {{infoctrl.selectedBeard.sub_title}}
             </h4>
             <p ng-repeat="para in infoctrl.selectedBeard.description">{{para}}</p>
         </div>
         <div class="beardInfoRightSection">
-            <div class="thumbnail">
-                <img ng-src="img/{{infoctrl.selectedBeard.image}}" alt="WeirdBeard">
+            <div class="thumbnail imgtile">
```
Comment (smandada) [inline public/beardinfo/beardinfo.html]: `img-title`

File: public/beardinfo/beardinfo.html [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+                <img class="infoimg" ng-src="img/{{infoctrl.selectedBeard.image}}" alt="WeirdBeard">
```
Comment (smandada) [inline public/beardinfo/beardinfo.html]: `info-img`

File: public/index.html [hunk 5]
```diff
@@ -24,7 +27,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>                        
               </button>
-          <a class="navbar-brand" ui-sref="home">WeirdBeard</a>
+          <a class="navbar-brand" ui-sref="home">𝖜𝖊𝖎𝖗𝖉𝖇𝖊𝖆𝖗𝖉</a>
```
Comment (smandada) [inline public/index.html]: Let's plan to use a google font in the next iteration, for now its fine.

Comment (Raghu-Ch) [inline public/index.html]: I will fix it

File: public/beardinfo/beardinfo.html (hunk 4 above)
Comment (Raghu-Ch) [inline public/beardinfo/beardinfo.html]: I will fix it.

File: public/beardinfo/beardinfo.html (hunk 3 above)
Comment (Raghu-Ch) [inline public/beardinfo/beardinfo.html]: Got it and I'll fix it

File: public/index.css (hunk 1 above)
Comment (Raghu-Ch) [inline public/index.css]: I will fix it.

File: public/index.css (hunk 2 above)
Comment (Raghu-Ch) [inline public/index.css]: I will fix it.

**Pull request merged**